### DISCUSSION TITLE
一直咳嗽不戴口罩的人

### DISCUSSION THREAD

**Discussion by DarsooK** [1101-GS4719-C/Python-Community-GS4719-C#2242] [General]
咳嗽時不戴口罩會增加飛沫傳播的風險，可能會將病菌或病毒傳播給周圍的人。飛沫中的病原體可以在空氣中傳播數公尺，尤其在通風不良的室內環境裡。為了保護自己和他人，特別是在公共場所或人多的地方，戴口罩是一種簡單有效的防護措施。此外，咳嗽時用手肘或紙巾遮住口鼻，可以減少病原體的傳播，也是禮貌的行為。我都會放幾個口罩在包裡，以預防在密閉環境，如：公車、教室，遇到一直咳嗽不戴口罩的人剛好坐在身旁。

**Comment by ed112601513** [comment]
我覺得生病咳嗽不戴口罩的人十分過分，我這學期和上學期的室友都有人有這個壞習慣，害我常常被傳染，輕微一點的可能小感冒，嚴重一點的直接傳染給我流感，請他戴口罩又不聽勸，這種情況也只能自己戴口罩保護好自己了。

**Comment by Boying12** [comment]
我個人會隨身攜帶幾個口罩，特別是在密閉環境中，比如公車或教室，這樣即使遇到旁邊有人咳嗽不戴口罩的情況，我也能及時將口罩戴上，保護自己。這不僅是對自己健康的保障，也是對他人健康的負責。希望大家都能共同遵守這些防護措施，讓我們的生活環境更加安全與健康！

**Comment by christaaah** [comment]
我也不喜歡咳嗽不戴口罩的人，這樣會造成感染的風險，我得話是只要感冒或疑似感冒的話都會戴著口罩，平常有時過敏嚴重的話也會戴著口罩出門，甚至自從疫情以來我出門都戴著口罩，但還是希望那些咳嗽的人可以自覺得戴起口罩。

**Comment by qiu0109** [comment]
我自己是覺得戴口罩很悶，而且在鼻塞時戴口罩又更吸不到空氣，所以在疫情快結束時就不再戴口罩。但是如果我真的感冒咳嗽，我會盡量避開人群並且咳嗽時用衣服遮住，不對著人打噴嚏或咳嗽。

**Comment by wcyeee** [comment]
我也覺得咳嗽不戴口罩的人很過分，顯然他們只想到自己戴口罩會不會不舒服，沒有想到旁邊的人會不會害怕被傳染。如果是我咳嗽的話一定會戴口罩，萬一真的沒有口罩我也會儘量忍住然後咳的時候衣袖遮掩。

**Comment by ShuliANN** [comment]
我覺得如果咳嗽，真的應該戴口罩，但若是手邊真的沒有口罩，也應該以衣物遮蔽，不能這麼自私沒有顧慮到其他人的健康，這是對別人的基本尊重！

**Comment by yue083139** [comment]
我個人是覺得戴口罩很不舒服的，所以在家裡不戴口罩，但是如果是公共場合的話，如果真的是感冒還是會拿口罩來戴，畢竟也不想傳染給別人，有時候如果是單純空氣太乾冷的話，我會盡量用衣物來遮蔽以免干擾到其他人，或者是忍住喝口水看會不會好點。

**Comment by Teddy111802518** [comment]
我是覺得戴口罩這件事情，有兩面，你沒生病就是保自己，生病了就是保護別人，但有人就是自私，咳嗽還不戴口罩，真的扯，不果我也有逾過那種連口罩都買不起的，好像也不能怪他，飯都吃不飽了，誰還管你什麼保護別人，不過至少咳嗽的時候可以手或衣服遮一下

**Comment by forestdew507** [comment]
隨身帶著幾個口罩我覺得是相對能保護自己的作法，( 沒忘記的話 ) 我也都會備幾個，人數眾多的密閉空間和附近有人咳嗽不戴口罩 ( 甚至座位就剛剛好在我正後面時 ) ，就能拿來保護自己。

另外就是酒精了 ! 雖然或許沒有 covid 疫情那麼嚴重，但隨身帶著小罐酒精也頗方便 ~ 以備不時之需。

**Comment by yuan3x2** [comment]
我認為這種人其實蠻自私的，如果是自己在家就算了，但外出工作或上學時就有可能傳給周遭的人，此外，當遇到疫情時，這種行為更是應該被唾棄，因為可能會傳染給免疫力較弱的人導致重病或死亡。
